### ISSUE TITLE
Intro text on some authorities is really long

### ISSUE THREAD

**Issue by henare** [openaustralia/righttoknow#651]
e.g. https://www.righttoknow.org.au/new/tasmanian_department_of_health_and_human_services

![selection_001](https://cloud.githubusercontent.com/assets/48945/19634806/8ed3317c-9a09-11e6-9813-f051bc16b14c.png)

In this case it's because we included all the hospital names as a hack so that when you search you find this authority. Maybe this needs to be behind a "More..." reveal on our theme?

**Comment by Keerti-Gautam** [comment]
Hi @henare,
I think it would be a good idea to list all the hospitals within a "See the list of Covered hospitals" link just after "... medical clinics throughout Tasmania" line. This, in my opinion, would increase the readability of the visitors of the website. What do you suggest?

**Comment by garethrees** [comment]
As a quick win, you could use a `<details>` element:

```html
<p>The Department of Health and Human Services provides a comprehensive, high quality, safe and sustainable health and human system for Tasmanians.</p> <details> <p><summary>It’s portfolio covers hospitals and medical clinics throughout Tasmania,</summary> including: Tasmanian Health Organisation North, Launceston General Hospital, St Helens District Hospital, Deloraine District Hospital, George Town District Hospital and Community Centre, North East Soldiers Memorial Hospital and Community Centre, Beaconsfield MPS, Campbell Town MPS, Flinders Island MPC, St Marys Community Health Centres, Cape Barren Island Nursing Centre, John L Grove Rehabilitation Unit, Mayne Street Day Centre, Ravenswood Community Health Centres, Westbury Community Health Centres, Northern Integrated Care Service, Kings Meadows Community Health Centres, Toosey Inc (Longford), Longford Community Health Centres, Tasmanian Health Organisation South, Royal Hobart Hospital, New Norfolk District Hospital, Swansea Community Health Centres, Midlands MPHC, Central Highlands Community Health Centres, Triabunna / Spring Bay Community Health Centres, Bridgewater / Brighton Community Health Centres, Glenorchy Community Health Centres, Clarence ICC, Sorell Community Health Centres, Clarence Plains / Rokeby Community Health Centres, Kingston Community Health Centres, Huon Community Health Centres, Cygnet Community Health Centres, Bruny Island Community Health Centres, Hobart Repatriation Centre, Tasmanian Health Organisation North West, North West Regional Hospital, Mersey Community Hospital, Smithton District Hospital, King Island District Hospital, West Coast District Hospital, Burnie Community Health Centres, Central Coast Community Health Centres - Ulverstone, Devonport Community Health Centres, James Muir Community Health Centres - Wynyard, Rosebery Community Health Centres, Strahan Community Health Centres, Zeehan Community Health Centres, King Island District Hospital and Health Centre, West Coast District Hospital, Correctional Primary Health Service, the Family Violence Counselling and Support Service. and then Disability and Community Service.</p> </details>
```

![screen shot 2017-03-27 at 10 02 18](https://cloud.githubusercontent.com/assets/282788/24348688/793dfb50-12d4-11e7-9bed-ea13c8174716.png)

**Comment by benrfairless** [comment]
Does that work with our current version of Alaveteli? Might need to ensure it's in the CSS (ping @equivalentideas).

Any change will need to be made in Admin console not via a pull request. 

--
Ben Fairless
Volunteer - Right to Know - @RightToKnowAu

A project of the OpenAustralia Foundation

**Comment by garethrees** [comment]
Yeah, I think the notes field has always rendered HTML. Might want to check browser support of `<details>` and compare to your user base (Google Analytics should show browser usage).

**Comment by equivalentideas** [comment]
@garethrees I really like that solution, and caniuse suggests our 70% of our users have support for `<details>`.

![screen shot 2017-03-28 at 12 01 59 pm](https://cloud.githubusercontent.com/assets/1239550/24384274/643b5efc-13ae-11e7-8a93-d9d548c4f97b.png)

@garethrees this does seem like something that might be best handled by the software, rather than error prone humans—but what do you recon? Is this something it might be worth automating upstream?

Thanks for getting this issue going again @Keerti-Gautam 👍 

> I think it would be a good idea to list all the hospitals within a "See the list of Covered hospitals" link just after "... medical clinics throughout Tasmania" line. This, in my opinion, would increase the readability of the visitors of the website. 

Do you mean with a little disclosure toggle like in @garethrees’s mock up here? https://github.com/openaustralia/righttoknow/issues/651#issuecomment-289394857

**Comment by garethrees** [comment]
> Is this something it might be worth automating upstream?

I think it would be quite tricky to add to Alaveteli, and unlikely to be something that we'd be able to prioritise soon. It wouldn't be obvious what to put in the `<summary>` and what should be hidden by default. We also have partners using the notes field for extra contact info and various other ways, so actually I think its better for this to be manual.

**Comment by equivalentideas** [comment]
>  so actually I think its better for this to be manual.

That makes sense, thanks @garethrees 👍

**Comment by benrfairless** [comment]
So if we are going to do this manually, there are 63 authorities on Right to Know that have more than 100 words in the details. 

These are they (in order of biggest words) - I'll go through these in the Admin console and see if the details tab will work.
- [x] Tasmanian Department of Health and Human Services
- [x] Federal Circuit Court of Australia
- [x] Australia-India Council
- [x] Museum of Australian Democracy
- [x] LINC Tasmania
- [x] Defence Imagery and Geospatial Organisation
- [x] Reserve Bank of Australia
- [ ] Australia-China Council
- [ ] Australian Fisheries Management Authority
- [ ] Gene Technology Ethics and Community Consultative Committee
- [ ] Australian Statistics Advisory Council
- [ ] ACTEW Corporation Limited
- [ ] International Air Services Commission
- [ ] Secretariesâ€™ ICT Governance Board
- [ ] Council on Australia Latin America Relations
- [ ] Department of Parliamentary Services
- [ ] Young and Well CRC
- [ ] Independent Expert Scientific Committee on Coal Seam Gas and Large Coal Mining Development
- [ ] Wound Management Innovation CRC
- [ ] Specialist Medical Review Council
- [ ] Tasmanian Victims Support Services
- [ ] Queensland Director of Public Prosecutions
- [ ] Housing And Community Services ACT
- [ ] SA Health
- [ ] WorkCover Tasmania
- [ ] Royal Australian Navy Relief Trust Fund
- [ ] Public Trustee For The ACT
- [ ] ACT City Services
- [ ] Queensland Curriculum and Assessment Authority
- [ ] Australian Taxation Office
- [ ] National Industrial Chemicals Notification and Assessment Scheme
- [ ] Australian Financial Security Authority
- [ ] Water Efficiency Labelling and Standards Regulator
- [ ] ACT Human Rights And Discrimination Commissioner
- [ ] CRC for Spatial Information
- [ ] Tasmanian Office of Consumer Affairs and Fair Trading
- [ ] Aboriginals Benefit Account Advisory Committee
- [ ] Queensland Civil and Administrative Tribunal
- [ ] Queensland Motor Accident Insurance Commission
- [ ] Commonwealth Grants Commission
- [ ] Superannuation Complaints Tribunal
- [ ] Australian Children's Education and Care Quality Authority
- [ ] Tasmanian Communities, Sport and Recreation
- [ ] Financial Reporting Council
- [ ] Infrastructure Australia
- [ ] Queensland Heritage Council
- [ ] Pathology Services Table Committee
- [ ] CRC for Aboriginal and Torres Strait Islander Health
- [ ] Invasive Animals CRC
- [ ] Legislative and Governance Forum on Gene Technology
- [ ] Fuel Standards Consultative Committee
- [ ] Tourism Quality Council of Australia
- [ ] Small Business Advisory Committee
- [ ] Future Farm Industries CRC
- [ ] Tasmanian Resource Management and Planning Appeal Tribunal
- [ ] Emissions Reduction Fund
- [ ] Queensland Building and Construction Industry (Portable Long Service Leave) Board
- [ ] High Court of Australia
- [ ] The Vision CRC
- [ ] National Offshore Petroleum Safety and Environmental Management Authority
- [ ] Forensic Science Service Tasmania
- [ ] Office of the Queensland Parliamentary Counsel

**Comment by benrfairless** [comment]
Details seems to be working pretty well:
<img width="658" alt="651-3" src="https://cloud.githubusercontent.com/assets/6841582/24609292/d8aca8d8-18ac-11e7-9e74-b3d906b00b0d.png">
<img width="661" alt="651-4" src="https://cloud.githubusercontent.com/assets/6841582/24609293/d8af41ce-18ac-11e7-9cdd-2afc08dbe9d2.png">

**Comment by Keerti-Gautam** [comment]
@benrfairless I see there are a lot of authorities here. I was searching for something which would automate this, so you wouldn't have to do this manually for every authority, but without any avail. Let me know if you find something that can be helpful! I'd be glad to know. Also, tell me if I can be of any help. :smiley:

**Comment by benrfairless** [comment]
Hey @Keerti-Gautam it might be possible to do via CSV - I got a CSV of the attached [here](https://www.righttoknow.org.au/body/all-authorities.csv), and did some excel magic to work out the number of words. I know that we can [upload CSVs to perform mass edits of documents](http://alaveteli.org/docs/running/admin_manual/#creating-changing-and-uploading-public-authority-data), this would save me having to update each authority individually.

**Comment by Keerti-Gautam** [comment]
OH! 
This is severely cool! :smile: Great!
Thanks a lot for telling me!
